test/basic.html

```diff
@@ -25,11 +25,20 @@
 
   <script>
 
-    window.localStorage.setItem('core-localstorage-test', '{"foo":"bar"}');
-    var storage = document.querySelector('#localstorage');
-    
+    var storage;
+
     suite('basic', function() {
 
+      before(function() {
+        window.localStorage.setItem('core-localstorage-test', '{"foo":"bar"}');
+        storage = document.querySelector('#localstorage');
+        storage.load();
```

Comment:
fixed. I changed core-localstorage initialization to .load() on ready(), and added an observers: for name + useRaw that reload if values change.